example/cavity/Fluid/system/controlDict

```diff
@@ -0,0 +1,53 @@
+FoamFile
+{
+    version     2.0;
+    format      ascii;
+    class       dictionary;
+    location    "system";
+    object      controlDict;
+}
+// * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * //
+
+application         pimpleFoam;
+
+startFrom           latestTime;
+
+startTime           0;
+
+stopAt              endTime;
+
+endTime             100;
+
+deltaT              0.05;
+
+writeControl        runTime;
+
+writeInterval       0.10;
+
+purgeWrite          0;
+
+writeFormat         ascii;
+
+writePrecision      6;
+
+writeCompression    off;
+
+timeFormat          general;
+
+timePrecision       0;
+
+runTimeModifiable   yes;
```

Comment:
The end time you set in OpenFOAM should not matter, the adapter should overwrite it. Could you maybe try setting `runTimeModifiable no;` and see if that helps?